airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java

```diff
@@ -95,6 +95,10 @@ public JsonNode toDatabaseConfigStatic(final JsonNode config) {
 
     additionalParameters.forEach(x -> jdbcUrl.append(x).append("&"));
 
+    if (config.has("jdbc_url_params")) {
+      jdbcUrl.append(config.get("jdbc_url_params").asText());
```

Comment:
The custom URL params cannot be directly appended. Parameters like `ssl=true` and `sslmode=require` should not be modified if they exist. You can reference the generic patten in [`AbstractJdbcDestination.java`](https://github.com/airbytehq/airbyte/blob/v0.36.0-alpha/airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/AbstractJdbcDestination.java#L97) to handle the custom JDBC params. The general idea is:
1. Get the default params.
2. Get the custom params.
3. Check that the custom params do not override the default params.
4. Merge the two params into one map.

Currently, this pattern is only used in `MySqlDestination` if I remember correctly. It can be copied to the abstract jdbc source probably so that it can be easily applied to other sources.

Reply:
Do you think we should rename this param based on that? Its not really JDBC Url Params based on how its implemented, its more like "JDBC Config Params" since were not appending to URL but just adding into the json config we use now. Guess its not any functionally different, but confusing language

Reply:
Agree that "JDBC config params" is more accurate. However, changing param names is usually tricky because it can break the connection for all existing users. Also this is not a big issue, since the param name is hidden from the user. So I don't think it has a high priority. We can wait until we support connection config migration in the future.